app/controllers/report.js

```diff
@@ -11,10 +11,16 @@ const selectedFillLayer = selectedFeatures.fill;
 const { service } = Ember.inject;
 const { alias } = Ember.computed;
 
+// 0 = city; 1..5 = boro
+const COMPARISON_GEOIDS = [0, 1, 2, 3, 4, 5]; // eslint-disable-line
```

Comment:
What is this?  It is never used.

Reply:
You're right, I t should be removed, it's just a list of the geoids that may be passed into the request for comparison